### DISCUSSION TITLE
Alias to multiple recipients?

### DISCUSSION THREAD

**Discussion by MarcS1975** [docker-mailserver/docker-mailserver#3805] [Q&A] [answered]
Is it possible to setup an alias so that 2 recipients receive all the emails simultaniously?
e.g. alias account1 **->** account2 and account3

**Comment by polarathene** [comment, marked as answer]
Did you try it?

```console
# Create some accounts:
$ setup email add john.doe@example.test bad-password
$ setup email add jane.doe@example.test bad-password

# Create the alias (or add a new recipient to an existing alias):
$ setup alias add my-alias@example.test john.doe@example.test
$ setup alias add my-alias@example.test jane.doe@example.test

# Show the aliases:
$ setup alias list | grep my-alias@example.test
* my-alias@example.test john.doe@example.test,jane.doe@example.test

# Config file it creates/updates:
$ cat /tmp/docker-mailserver/postfix-virtual.cf
my-alias@example.test john.doe@example.test,jane.doe@example.test
```

Note that this won't be supported by Dovecot Quotas (`ENABLE_QUOTAS=1`), some other features may also only expect to work with aliases that map only to a single account directly (_eg: no alias to alias to account_).

We presently don't allow for aliases and accounts to share the same address/value. This isn't a feature you'd likely receive any support on from maintainers if you do have any problems related to it.

**Comment by MarcS1975** [comment]
I haven’t tried it because I didnt want to mess up my email accounts. I
thought it may be a straight answer.
Quotas is not relevant for me.

**Reply by polarathene** [reply]
> I didnt want to mess up my email accounts

You could just run another local DMS instance?

```bash
# Basic DMS container (_it will expect a user added within 2 minutes of starting or it will shutdown_):
docker run --rm -it --hostname mail.example.test --name dms mailserver/docker-mailserver

# Then run bash shell in DMS where you can use `setup` CLI:
docker exec -it dms bash
setup ...
```

No need to mess with your existing setup to try this sort of thing.

**Comment by MarcS1975** [comment]
Ok, I'll check it out. Thanks

**Comment by MarcS1975** [comment]
i confirm that alias to 2 recipients works. 
**BTW the spam filter** also works brilliantly out of the box !!! I have not changed any settings and even tricky emails that are hard to detect for humans are well identified as SPAM and delivered to the SPAM folder of my mail client. Thank you and Well done!

**Reply by polarathene** [reply]
> **BTW the spam filter** also works brilliantly out of the box !!!

Did you choose Amavis+SpamAssassin or Rspamd?

**Comment by MarcS1975** [comment]
I have enable_spamassesin=1

**Comment by MarcS1975** [comment]
does your Rspamd package come with the WebGUI?

**Reply by polarathene** [reply]
Yes, [view the docs](https://docker-mailserver.github.io/docker-mailserver/v13.3/config/security/rspamd/#web-interface).

The rspamd feature is still WIP, I have heard it's quite good but the docs may still need some extra revision to be easier for users. It is intended to eventually become the default and replace SpamAssassin.

**Comment by MarcS1975** [comment]
Looks awesome and seems very user friendly.
Then the only thing that’s missing for DMS is an easy way for email users
to change their own password.

**Reply by polarathene** [reply]
> Then the only thing that’s missing for DMS is an easy way for email users to change their own password.

Yes, quite a few requests for that. Not enough contributors to get momentum there though.

We do have oauth2 support now, so technically you could use that (still requires an account added to DMS). Or if you can use LDAP with some way to manage passwords separately there.
